### PR TITLE
Title for future me

### PR DESCRIPTION
**Pull request by intheswim**
__SIGNED_FOR_FUTURE__

### CODE REVIEW

**Comment by intheswim** [comment]
@beardofedu Please approve.

**Comment by intheswim** [comment]
closes #8

**Comment by intheswim** [comment]
@epitrochoider please review

File: _posts/0000-01-03-intheswim.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "intheswim's slide one"
```
Comment (epitrochoider) [inline _posts/0000-01-03-intheswim.md:3]: This looks great

File: _posts/0000-01-03-intheswim.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
```
Comment (epitrochoider) [inline _posts/0000-01-03-intheswim.md:3]: I'm glad you got rid of this old title

File: _posts/0000-01-03-intheswim.md [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
 
 ![angry-cat](https://cloud.githubusercontent.com/assets/16547949/25400569/064d7bc6-29c1-11e7-90dc-fef44edd9f0a.jpg)
 {: .center}
 
-Caption here
+**TEXT HERE**
+
+_text here_
+
+An angry cat.
```
Comment (epitrochoider) [inline _posts/0000-01-03-intheswim.md]: 👍

**Review by epitrochoider — APPROVED** [review]

**Pull request merged**